README.md

```diff
@@ -46,11 +22,11 @@ To run the *baseline* tracking configuration you can derive it from the simplifi
 ```
 cmsDriver.py Phase2 -s HLT:75e33 \
     --processName=HLTX \
-     --conditions 131X_mcRun4_realistic_v5  \
+     --conditions auto:phase2_realistic_T21  \
```

Comment:
production was running with `--conditions 131X_mcRun4_realistic_v5` , as e.g. in [TTbar sample](https://cms-pdmv.cern.ch/mcm/public/restapi/requests/get_setup/TRK-Phase2Spring23DIGIRECOMiniAOD-00001)

I see that [autoCondPhase2](https://github.com/cms-sw/cmssw/blob/CMSSW_13_1_0/Configuration/AlCa/python/autoCondPhase2.py) some of the payload parameters are overridden. 
@mmusich @AdrianoDee can this introduce some problems?

Reply:
T21 (and derived) is the default, so normally the physical GT for sample production should (payload per payload) coincide with the symbolic `auto` key

Reply:
The reason why I switched to that (I didn't investigate myself):
https://twiki.cern.ch/twiki/bin/viewauth/CMS/HighLevelTriggerPhase2SimplifiedMenu